index.ts

```diff
@@ -112,20 +118,28 @@ export function encodeRunestoneUnsafe(runestone: RunestoneSpec): Buffer {
   }));
 
   let etching: Option<Etching> = None;
+  let etchingCommitment: string | undefined = undefined;
   if (runestone.etching) {
     const etchingSpec = runestone.etching;
-
-    if (!etchingSpec.rune && etchingSpec.spacers?.length) {
-      throw Error('Spacers specified with no rune');
```

Comment:
You don't need this cuz spacers are inferred from SpacedRunes constructor which does the needed validations